fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,3 @@
+<i>lichess</i> is built for the love of chess. The chess server has 150 000 individual users daily and growing fast. Several chess variants are supported (Crazyhouse, Chess 960, King Of The Hill, Three-check, Antichess, Atomic chess, Horde, Racing Kings), either in online and offline modes!
+
+For screenshots, please visit the project’s Github presence. For help and more details, see its website.
```

Comment:
Not sure about that one.

```suggestion
For screenshots, please visit the project’s GitHub page. For help and further details, visit the website.
```

Reply:
I picked "presence" as it seemed to be the most correct one: It's more than a page, but not your site – and "Github pages" would be ambiguous. "Github project" on the other hand would sound funny in the context ("the project's Github project"). But up to you.

Considering that screenshots are meanwhile included with the Fastlane structures (they were not when that description was created), the first sentence could also be dropped altogether, just leaving

```suggestion
For help and more details, please visit the project's website.
```

This technically even includes "screenshots" with the term "more details" :wink: